specification/columns/tags.md

```diff
@@ -9,7 +9,9 @@ The Tags column adheres to the following requirements:
 * The Tags column MUST contain user-defined and provider-defined tags.
 * The Tags column MUST only contain finalized tags.
 * The Tags column MUST be in [Key-Value Format](#key-valueformat).
-* A Tag key without a specified value MUST have its tag value set to null.
+* A Tag key with a non-null value for a given resource SHOULD be included in the tags column.
+* Depending on the provider's tag finalization process, a Tag key with a null value for a given resource MAY be included in the tags column.
+* A Tag key that does *not* support a corresponding value, sometimes referred to as a *label*, MUST have a corresponding true (boolean) value set.
```

Comment:
Can we make sure we have an ack from practitioners who use labels that this is desirable? Just want to confirm a Boolean value is what they want/expect.

Reply:
We discussed in the call and had a small ack. If anyone is concerned, please leave a comment or create a new issue.

Reply:
Michael said at the maintainer meeting he's good with the logic that this is following. Resolving.